test/meadow/data/works_test.exs

```diff
@@ -465,4 +465,24 @@ defmodule Meadow.Data.WorksTest do
       end)
     end
   end
+
+  describe "mint_ark/1" do
+    setup %{work_type: work_type} do
+      {:ok, work: work_fixture(%{work_type: %{id: work_type, scheme: "work_type"}})}
+    end
+
+    seed_values("coded_terms/work_type")
```

Comment:
This bit reads the valid `work_type`s from the JSON seed file and creates a separate test per work type. There are two reasons for doing it this way (as opposed to iterating over `CodedTerms.list_coded_terms(:work_type)`:
1. Easier to read the output in case something fails (because the work type will be in the test name)
2. Can't iterate over the `CodedTerms` table outside the test block because the sandbox DB connection isn't checked out yet at that point